joplin/css/admin.scss

```diff
@@ -0,0 +1,78 @@
+.full input {
```

Comment:
This is pretty much just a rename of our admin.css, but since it's been moved and renamed GH seems to think it's new